src/System.Xml.XmlSerializer/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -368,7 +368,7 @@ private void WriteMemberElementsIf(ref object o, CollectionMember collectionMemb
             }
             else
             {
-                if (anyElementMember!= null & anyElementMember.Mapping != null)
+                if (anyElementMember!= null && anyElementMember.Mapping != null)
```

Comment:
Nit: Add a space between `anyElementMember` and `!=`, while making changes to this line?